cosmos-init/plugins/packages/cosmosc2-tool-admin/src/tools/CosmosAdmin/tabs/PluginsTab.vue

```diff
@@ -22,42 +22,37 @@
     <v-row no-gutters align="center" class="px-2">
       <v-col>
         <v-file-input
-          v-model="files"
-          multiple
+          v-model="file"
           show-size
           accept=".gem"
           class="mx-2"
-          label="Click to select plugin .gem file(s)"
+          label="Click to select plugin .gem file to install"
```

Comment:
New method requires a single upload at a time? I don't think that's a bad thing, just wondering.

Reply:
Yes, it was too complicated with multiple gems and dealing with plugin.txt + variables